base/applications/sdbinst/sdbinst.c

```diff
@@ -0,0 +1,880 @@
+/*
+ * PROJECT:     ReactOS sdbinst
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Application compatibility database installer
+ * COPYRIGHT:   Copyright 2020 Max Korostil (mrmks04@yandex.ru)
+ */
+
+#include <tchar.h>
+
+#include <windef.h>
+#include <winbase.h>
+#include <winreg.h>
+#include <strsafe.h>
+#include <objbase.h>
+#include <apphelp.h>
+
+#define APPCOMPAT_CUSTOM_REG_PATH L"Software\\Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\Custom"
+#define APPCOMPAT_LAYERS_REG_PATH L"Software\\Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\Layers"
```

Comment:
On server 2003 it seems that layers are installed under `AppCompatFlags\Custom\Layers`

Reply:
I think you right